pkg/helm/testdata/schema.json

```diff
@@ -29,10 +29,7 @@
               "type": "boolean"
             },
             "set": {
-              "type": "object",
-              "additionalProperties": {
-                "type": "string"
-              }
+              "type": "object"
```

Comment:
I think you may need to keep this as `"additionalProperties": true` so that intellisense in VSCode allows anything to be typed in under `set`. But you should verify that in VSCode yourself.